src/makeCalculator.js

```diff
@@ -4,7 +4,51 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(currentValue, value) {
+      return currentValue + value;
+    },
+
+    subtract(currentValue, value) {
+      return currentValue - value;
+    },
+
+    multiply(currentValue, value) {
+      return currentValue * value;
+    },
+
+    divide(currentValue, value) {
+      return currentValue / value;
```

Comment:
Consider using the assignment operator `/=` to divide the current value by the given value. This will simplify the expression and follow the code style best practices.